docs/modules/ROOT/pages/erc20.adoc

```diff
@@ -2,7 +2,7 @@
 
 :fungible-tokens: https://docs.openzeppelin.com/contracts/4.x/tokens#different-kinds-of-tokens[fungible tokens]
 :eip20: https://eips.ethereum.org/EIPS/eip-20[EIP-20]
-:version: https://github.com/OpenZeppelin/cairo-contracts/releases/tag/v0.7.1[Contracts v0.7.1]
+:version: https://github.com/OpenZeppelin/cairo-contracts/releases/tag/v0.8.0-beta.1[Contracts v0.8.0-beta.1]
```

Comment:
```suggestion
:version: https://github.com/OpenZeppelin/cairo-contracts/releases/tag/v0.7.0[Contracts v0.7.0]
```
This shouldn't change because it's referencing when the breaking change in the decimals design occurred